com.netxms.Client.metainfo.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+   <id>com.netxms.Client</id>
+   <metadata_license>CC-BY-SA</metadata_license>
+   <name>NetXMS</name>
+   <summary>NetXMS Client</summary>
+   <description>
+      <p>Desktop client for NetXMS - network monitoring system.</p>
+   </description>
+   <url type="homepage">https://netxms.com</url>
+   <url type="bugtracker">http://track.radensolutions.com/</url>
+   <url type="help">http://netxms.org/documentation/adminguide/</url>
+   <launchable type="desktop-id">com.netxms.Client.desktop</launchable>
+   <releases>
+      <release version="4.5.5" date="2024-04-02">
+         <url>https://github.com/netxms/changelog/blob/master/ChangeLog.md</url>
+         <description>
+            - Fixed scheduled file upload
```

Comment:
this should be a list. Otherwise it's rendered as a single paragraph.

Reply:
https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#description